user-registration-app/src/app/services/AuthService/auth.service.spec.ts

```diff
@@ -1,21 +0,0 @@
-import { TestBed, inject } from '@angular/core/testing';
```

Comment:
The spec files are tests automatically generated when components are created. Seeing as how there are no tests, there's not much point keeping these